package-lock.json

```diff
@@ -168,6 +167,11 @@
         "troika-three-text": "^0.45.0"
       }
     },
+    "node_modules/@types/hubs/node_modules/bitecs": {
+      "version": "0.3.38",
+      "resolved": "git+ssh://git@github.com/mozilla/bitECS.git#913b4ae261684eee205251e41a3250d1c1a2817e",
```

Comment:
Not sure why this still resolved to the old repo. 🤔

Reply:
@Exairnous I think I've resolved this now.

Reply:
Thanks.